BedrockServer.cpp

```diff
@@ -192,10 +192,10 @@ void BedrockServer::sync()
                                                             args["-peerList"], args.calc("-priority"), firstTimeout,
                                                             _version, args.test("-parallelReplication")));
 
-    // This should be empty anyway, but let's make sure.
-    if (_completedCommands.size()) {
-        SWARN("_completedCommands not empty at startup of sync thread.");
-    }
+    // Control port and not command port so that followers can still escalate to leader when leader's command port is
+    // closed.
+    // TODO: Pass this in the constructor and make it const?
```

Comment:
Can we do this in this PR?

Reply:
It changes the signature of the constructor so that everything that uses it (which I believe includes auth tests) also needs to change, so I wanted to skip it for now to keep this change as focused as possible.